api4/team_local.go

```diff
@@ -126,7 +128,14 @@ func localInviteUsersToTeam(c *Context, w http.ResponseWriter, r *http.Request)
 		if len(goodEmails) > 0 {
 			err := c.App.Srv().EmailService.SendInviteEmails(team, "Administrator", "mmctl "+model.NewId(), goodEmails, *c.App.Config().ServiceSettings.SiteURL)
 			if err != nil {
-				c.Err = err
+				switch {
+				case errors.Is(err, email.NoRateLimiterError):
```

Comment:
Any reason we are not printing the error here too?

Reply:
Answered below.